src/server/api/root.ts

```diff
@@ -1,11 +1,14 @@
 import { createTRPCRouter } from "~/server/api/trpc";
+import { companyRouter } from "./routers/company";
 
 /**
  * This is the primary router for your server.
  *
  * All routers added in /api/routers should be manually added here.
  */
-export const appRouter = createTRPCRouter({});
+export const appRouter = createTRPCRouter({
+  companyRouter,
```

Comment:
Passing in a key into the object that is passed to `createTRPCRouter` makes API more intuitive. For instance, if we were to call one of the endpoints (such as the `getAll` endpoint), the call would be `api.companyRouter.getAll.query()` instead of the _arguably_ more idiomatic `api.company.getAll.query()`. Try something like the code below.

```javascript
export const appRouter = createTRPCRouter({
  company: companyRouter,
});

```